services/visual-studio-app-center/visual-studio-app-center-releases-size.service.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const prettyBytes = require('pretty-bytes')
+const { nonNegativeInteger } = require('../validators')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+
+const schema = Joi.object({
+  size: nonNegativeInteger.required(),
+}).required()
+
+module.exports = class VisualStudioAppCenterReleasesSize extends BaseVisualStudioAppCenterService {
+  static get category() {
+    return 'size'
+  }
+
+  static get route() {
+    return {
+      base: 'visual-studio-app-center/releases/size',
+      pattern: ':owner/:app/:token',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Visual Studio App Center Size',
+        namedParams: {
+          owner: 'jct',
+          app: 'my-amazing-app',
+          token: 'ac70cv...',
+        },
+        staticPreview: this.render({ size: 8368844 }),
+        keywords,
+        documentation,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'size',
+      color: 'blue',
+    }
+  }
+
+  static render({ size }) {
+    return {
+      // AppCenter displays in mebibytes.
```

Comment:
Are you sure that the value in the API response is not just in bytes? Based on the [swagger spec](https://api.appcenter.ms/preview/swagger.json), `size` seems to be bytes:

```json
...
        "size": {
          "description": "The release's size in bytes.",
          "type": "number"
        },
...
```

The AppCenter UI may display the value in the mebibyte format, but that doesn't necessarily mean the API response uses that format. If you have any other info indicating that the API is mebibytes, then let's include URLs to that documentation in an inline comment

Reply:
Sorry about that. I know that all UI elements display it in mebibytes, but I don't have any sources for the API also being in mebibytes. I'm currently just converting the response to match what the UI displays.

Reply:
Gotcha, I thought that was the case. I don't think we need to be doing that conversion. We're using `pretty-bytes` for all our size related badges, which uses base 10. AFAICT the API response is in bytes which we can (and should) pass directly into `pretty-bytes` for formatting so that this is consistent with Shields formatting for sizes